teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>Can we drop the email here? (--email m.gorostiola.gonzalez@lacdr.leidenuniv.nl)</p><p>I might mean you'll get an email whenever someone runs the code, no?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>I asked this in the PR. We will need a mock gmail account. Do you have a preference for the name?</p>

Reply:
<p>Also, I have modified the client code to allow for one extra argument (--outdir) so we don't have to change directory before and after the query. </p>

Reply:
True, I saw that PR question only later, sorry - @AndreaVolkamer how would you like to proceed here?